.circleci/config.yml

```diff
@@ -78,6 +78,13 @@ jobs:
           mkdir -p $GRPC_DEST
           git clone --branch v1.2.0 https://github.com/grpc/grpc-go $GRPC_DEST
 
+    - run:
+        name: Vendor redis v6.15.3
+        # This step vendors go-redis/redis v6.15.3 inside our redis contrib
+        # to allow running the tests against the correct version of
+        # the redis library.
+        command: |
+          git clone --branch v6.15.3 https://github.com/go-redis/redis contrib/go-redis/redis/vendor/github.com/go-redis/redis
```

Comment:
Can we use the same environment variable as for grpc? I find it makes it more readable.

Reply:
Updated this to be consistent but also not too wide.